tests/core/test_views.py

```diff
@@ -0,0 +1,159 @@
+import pytest
+from django.test import TestCase, Client
+from .factories import UserFactory
+from django.contrib.auth.models import Group
+from django.http import HttpResponseRedirect, HttpResponse
+
+
+class TestCoreViews(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_home_view_is_resolved(self):
+        response = self.client.get('/')
+        self.assertEquals(response.status_code, 200)
+        self.assertTemplateUsed(response, 'index.html')
+
+    def test_about_view_is_resolved(self):
+        response = self.client.get('/about', {}, True)
+        self.assertEquals(response.status_code, 200)
+        self.assertTemplateUsed(response, 'core/about.html')
+
+    def test_community_view_is_resolved(self):
+        response = self.client.get('/community', {}, True)
+        self.assertEquals(response.status_code, 200)
+        self.assertTemplateUsed(response, 'core/community.html')
+
+    def test_support_versioning_view_is_resolved(self):
+        response = self.client.get('/support/versioning', {}, True)
+        self.assertEquals(response.status_code, 200)
+        self.assertTemplateUsed(response, 'support/versioning.html')
+
+    def test_fallback_views(self):
+        response = self.client.get('/doesnt_exist/')
+        self.assertEquals(response.status_code, 404)
+        self.assertTemplateUsed('404.html')
+
+    def test_fallback_views_unexpected_character(self):
+        response1 = self.client.get('/example../')
+        self.assertEquals(response1.status_code, 404)
+        self.assertTemplateUsed('404.html')
+        response2 = self.client.get('/!example/')
+        self.assertEquals(response1.status_code, 404)
+        self.assertTemplateUsed('404.html')
+
+
+class SetUpAdminViews(TestCase):
+    def setUp(self):
+        self.admin_client = Client()
+        self.user_client = Client()
+        self.staff_client = Client()
+        self.admin_user = UserFactory.build(username='admin', is_superuser=True)
+        self.admin_user.set_password('admin')
+        self.admin_user.save()
+        admin_login = self.admin_client.login(username='admin', password='admin')
+        self.assertTrue(admin_login)
+        self.staff_user = UserFactory.build(username='staff', is_staff=True)
```

Comment:
why are you using `UserFactory.build` and `UserFactory.create` sometimes?
always use `UserFactory.create`